apps/retrospective/services/sprint.go

```diff
@@ -45,6 +64,19 @@ func (service SprintService) DeleteSprint(sprintID string) (int, error) {
 		utils.LogToSentry(err)
 		return http.StatusInternalServerError, errors.New("failed to get sprint")
 	}
+
+	// Check if the sprint is deleteable.
+	is_sprint_deleteable, err := service.isSprintDeletable(sprint)
```

Comment:
use lower camel case naming convention for `is_sprint_deleteable`.

Also, `deletable` sounds better than `deleteable`.